src/node_modules/_file-explorer.js

```diff
@@ -0,0 +1,292 @@
+      return {
+        children: subtree instanceof Array ? subtree.map((item, index) => ({
+          key: index, value: item
+        })) : subtree instanceof Object ? Object.keys(subtree).map(key => ({
+          key: key, value: subtree[key]
+        })) : undefined
+      }
+    },
+    formatSelf: function (key, data) {
+      return yo`<label class=${data.children ? css.folder : css.file}
+        onload=${function (el) { adaptEnvironment(el, focus, hover) }}
+        onunload=${function (el) { unadaptEnvironment(el, focus, hover) }}
+        onclick=${editModeOn}
+        onkeydown=${editModeOff}
+        onblur=${editModeOff}
+      >${key}</label>`
+    }
+  })
+
+  container.appendChild(tv.render(data))
+
+  setTimeout(function () { // infere first active element from tab bar
+    var activePath = document.querySelector('.file.active').firstChild.innerText
+    var el = getElement(activePath)
+    focus.call(el,{})
+  },0)
+
+  var api = { }
+
+  return api
+  /******************************************************************************
+    HELPER FUNCTIONS (per instance)
+  ******************************************************************************/
+  function focus (event) {
+    event.cancelBubble = true
+    if (focusElement === this) return
+    if (focusElement) focusElement.classList.toggle(css.hasFocus)
+    focusElement = this
+    focusElement.classList.toggle(css.hasFocus)
+    var filepath = getPath(focusElement).file
+    if (filepath) files.event.trigger('fileFocus', [filepath])
+  }
+  function hover (event) {
+    if (event.type === 'mouseout') {
+      var exitedTo = event.toElement || event.relatedTarget
+      if (this.contains(exitedTo)) return
+      return this.removeChild(deleteButton)
+    }
+    this.appendChild(deleteButton)
+  }
+  function getPath (el) { return getPathRoot(container, el) }
+  function getElement (path) { return getElementRoot(container, path) }
+  function deletePath (event) {
+    event.cancelBubble = true
+    var el = this.parentElement.parentElement
+    var path = getPath(el)
+    if (confirm(`
+      Do you really want to delete "${path.file||path.folder}" ?
+      ${path.folder ? '(and all contained files and folders)' : ''}
+    `)) {
+      path = path.file || path.folder
+      el.parentElement.removeChild(el)
+      removeSubtree(files, path)
+    }
+  }
+  function editModeOn (event) {
+    var classes = this.parentElement.parentElement.parentElement.className
+    if (~classes.indexOf('hasFocus') && !this.getAttribute('contenteditable')) {
+      textUnderEdit = this.innerText
+      this.setAttribute('contenteditable', true)
+      this.classList.add(css.rename)
+      this.focus()
+    }
+  }
+  function editModeOff (event) {
+    if (event.type === 'blur' || event.which === 27 || event.which === 13) {
+      if (event.which === 13) {
+        event.preventDefault()
+        save = true
+      } else if (textUnderEdit !== this.innerText) var save = confirm(`
```

Comment:
standard is ok with how it's currently written - it's all standardjs compliant

...but no problem,

template literals can also be a string.

yes ok can do that to with the brackets...